.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'prettier/prettier': [
-      'error',
+      'warn',
```

Comment:
The update in the vue-cli created a bunch of prettier errors that broke our build.  This switches them to 'warn' so that we can still build the ui and add the update.